{
  "info": {
    "author": "Daniel Perrefort",
    "author_email": "",
    "bugtrack_url": null,
    "classifiers": [
      "Natural Language :: English",
      "Operating System :: OS Independent",
      "Programming Language :: Python",
      "Programming Language :: Python :: 3",
      "Programming Language :: Python :: 3.6",
      "Programming Language :: Python :: 3.7",
      "Programming Language :: Python :: 3.8",
      "Programming Language :: Python :: 3.9"
    ],
    "description": "",
    "description_content_type": "",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "",
    "keywords": "Django HTTP Error Pages",
    "license": "GPL v3",
    "maintainer": "",
    "maintainer_email": "",
    "name": "django-easy-error",
    "package_url": "https://pypi.org/project/django-easy-error/",
    "platform": "",
    "project_url": "https://pypi.org/project/django-easy-error/",
    "project_urls": null,
    "release_url": "https://pypi.org/project/django-easy-error/1.0.0/",
    "requires_dist": [
      "django"
    ],
    "requires_python": ">=3.6",
    "summary": "Easy template integration for django error pages",
    "version": "1.0.0",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 14062760,
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "24b9b034db6b435df0e369f71688d8843222f64aaf2ce724cb621cc88900bd7c",
        "md5": "90af7251a9598c70a3c92a3c000fac6b",
        "sha256": "02dd361f0bdb2f603eae817e81e966e501d19cc77a2df5b4c7983c1d1b971a38"
      },
      "downloads": -1,
      "filename": "django_easy_error-1.0.0-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "90af7251a9598c70a3c92a3c000fac6b",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": ">=3.6",
      "size": 16983,
      "upload_time": "2022-01-23T03:08:57",
      "upload_time_iso_8601": "2022-01-23T03:08:57.181746Z",
      "url": "https://files.pythonhosted.org/packages/24/b9/b034db6b435df0e369f71688d8843222f64aaf2ce724cb621cc88900bd7c/django_easy_error-1.0.0-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    },
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "f3243fc3ef19b8d3890d5d7e4fa2a91bd2b5f19e8a6c3d9d146a0f0589d531b2",
        "md5": "cddfe72295f08ca85fba55f7eec72a27",
        "sha256": "06c53629c91f42ef09cc66fa05dac29d6b01d54ef1e9b52bd6cd79be052e5c72"
      },
      "downloads": -1,
      "filename": "django-easy-error-1.0.0.tar.gz",
      "has_sig": false,
      "md5_digest": "cddfe72295f08ca85fba55f7eec72a27",
      "packagetype": "sdist",
      "python_version": "source",
      "requires_python": ">=3.6",
      "size": 16311,
      "upload_time": "2022-01-23T03:08:58",
      "upload_time_iso_8601": "2022-01-23T03:08:58.787520Z",
      "url": "https://files.pythonhosted.org/packages/f3/24/3fc3ef19b8d3890d5d7e4fa2a91bd2b5f19e8a6c3d9d146a0f0589d531b2/django-easy-error-1.0.0.tar.gz",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}